src/modules/fw_att_control/FixedwingAttitudeControl.hpp

```diff
@@ -230,7 +236,19 @@ class FixedwingAttitudeControl final : public ModuleBase<FixedwingAttitudeContro
 	ECL_YawController		_yaw_ctrl;
 	ECL_WheelController		_wheel_ctrl;
 
-	void control_flaps(const float dt);
+	/**
+	 * @brief Set current time delta for flap control (used in slew rate limiter)
+	 *
+	 * @param dt Current time delta [s]
+	 */
+	void controlFlaps(const float dt);
+
+	/**
+	 * @brief Set current time delta for spoiler control (used in slew rate limiter)
+	 *
+	 * @param dt Current time delta [s]
+	 */
+	void controlSpoilers(const float dt);
```

Comment:
Could we make the brief say something like "update flap (spoiler) control setting"? as it is not only setting the time delta, but also changing the commanded flap / spoiler value.

Reply:
yes ofc, that was a stupid copy&paste thingy.

Reply:
https://github.com/PX4/PX4-Autopilot/pull/19329/commits/3dd4403463f210743ac80a5d838a7de88f934259